README.md

```diff
@@ -1,17 +1,54 @@
 # pokeemerald-expansion
 
-### Important: DO NOT use GitHub's "Download Zip" option. Using this option will not download the commit history required to update your expansion version or merge other feature branches. Instead, please read [this guide](https://github.com/Pawkkie/Team-Aquas-Asset-Repo/wiki/The-Basics-of-GitHub) to learn how to fork the repository and clone locally from there.
+pokeemerald-expansion is ***NOT a playable romhack. It is a romhack base*** based off pret's [pokeemerald](https://github.com/pret/pokeemerald) decompilation project. It has multiple features available to romhackers so that they can create their own games, but it's not meant to be played on its own.
 
-## What is pokeemerald-expansion?
-
-pokeemerald-expansion is a decomp hack base project based off pret's [pokeemerald](https://github.com/pret/pokeemerald) decompilation project. It's recommended that any new projects that plan on using it, to clone this repository instead of pret's vanilla repository, as we regurlarly incorporate pret's documentation changes. This is ***NOT*** a standalone romhack, and as such, most features will be unavailable and/or unbalanced if played as is.
+## Should I use this or vanilla pokeemerald for my hack?
+The main advantage of using vanilla pokeemerald as a base is being able to link with other official GBA Pokémon games for battles and trading, pokeemerald-expansion can battle and trade with itself out of the box. If you don't mind losing full vanilla compatiblitity, we recommend using pokeemerald-expansion. Otherwise, use pret's pokeemerald. You'll still receive documentation improvements from pret, as we regurlarly incorporate pret's documentation changes.
 
 If you use pokeemerald-expansion in your hack, please add RHH (Rom Hacking Hideout) to your credits list. Optionally, you can list the version used, so it can help players know what features to expect.
```

Comment:
I think this should get a heading, something like

## Using pokeemerald-expansion

That goes from "If you use pokeemerald-expansion in your hack" to the start of the "Important:" PSA. I think we should include the sentence
`Please follow the instructions in `INSTALL.md` to get pokeemerald-expansion set up on your machine.`
at the end of this section, as per #5643 (and then I'll close that PR).

Reply:
Ignore this feedback, PR'd separately in #5720